redirect.md

```diff
@@ -25,13 +25,11 @@ Method modification:
 
 ```patch
   public void target() {
-+     double result;
```

Comment:
Is moving this out into a seperate method necessary? The readme explicitly says

> Note: the method modifications listed in these documents are what the code_ effectively _does, not what the modified bytecode will actually look like. Mixin generates additional methods in the target classes that are inlined here.

so that I don't have to do this (so far)

Reply:
oh